src/app/views/censusdata/create.html

```diff
@@ -0,0 +1,171 @@
+                <div class="col-md-9">      
+                <div class="checkbox">
+                    <label ng-repeat="tide in tideInfo"><input type="checkbox" value="{{tide.Id}}"> {{ tide.Description }} &nbsp;</label>        
+                </div>
+                </div>    
+            </div>  
+            <div class="form-group">
+                <label class="control-label col-md-3">Weather:</label>
+                <div class="col-md-9">
+                <div class="checkbox">
+                    <label ng-repeat="weather in weatherInfo"><input type="checkbox" value="{{weather.Id}}"> {{ weather.Description }} &nbsp;</label>      
+                </div>
+                </div>    
+            </div> 
+            <div class="form-group">
+                <label class="control-label col-md-3" for="observers">Observer(s):</label>
+                <div class="col-md-7">
+                <input type="text" class="form-control" id="observers" placeholder="Enter Observer(s) Name(s)">
+                </div>    
+            </div> 
+            <div class="form-group">
+                <label class="control-label col-md-3">Survey Vantage Point:</label>
+                <div class="col-md-9">
+                <div class="checkbox">
+                    <label ng-repeat="vantagePoint in vantagePointInfo"><input type="checkbox" value="{{vantagePoint.Id}}"> {{vantagePoint.Description}} &nbsp; </label>
+                </div>
+                </div>    
+            </div> 
+            <div class="form-group">
+                <label class="control-label col-md-3">Survey Access Point:</label>
+                <div class="col-md-9">
+                <div class="checkbox">
+                    <label ng-repeat="accessPoint in accessPointInfo"><input type="checkbox" value="{{accessPoint.Id}}"> {{accessPoint.Description}} &nbsp;</label>       
+                </div>
+                </div>    
+            </div>       
+            <div class="form-group">        
+                <div class="col-md-12">
+                <!--<div class="form-group panel panel-primary">
+                    <div class="panel-heading"></div>
+                    <div class="panel-body">-->
+                
+                <table class="table table-condensed">
+                    <thead>
+                    <tr class="info">
+                        <th>Species Name</th>
+                        <th>No. of Adults</th>
+                        <th>No. of Juvenile</th>
+                        <th>Behavior: Primary Activity</th>
+                        <th>Behavior: Secondary Activity</th>
+                        <th>Location/ Habitat Type</th>
+                        <th>% Feeding Success Rate</th>
+                        <th></th><th></th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td><input type="text" class="form-control" placeholder="Species Name"></td>
```

Comment:
As mentioned in the Slack comments, eventually we'll want to create one row for each bird species that can be used in the survey. We will just print the bird name and the user won't have to retype it.